lib/normalizer.c

```diff
@@ -1892,34 +1911,40 @@ grn_nfkc_normalize_unify_katakana_trailing_o(grn_ctx *ctx,
   return current;
 }
 
+typedef bool
+(*grn_nfkc_normalize_is_target_char_func)(const unsigned char *utf8_char,
+                                          size_t length);
+
+typedef struct {
+  grn_nfkc_normalize_is_target_char_func is_prolonged_sound_mark_like_char;
```

Comment:
Hmm. This may be redundant. We may be use more shorter name without reducing readability because struct name has `prolonged_sound_mark_like` information: 

```suggestion
  grn_nfkc_normalize_is_target_char_func is_target_char;
```

Reply:
Make sense.
Fixed.